CONTRIBUTING.md

```diff
@@ -21,28 +21,28 @@ For every pull request to `master` with automated tests you can check the logs o
 ## API
 <div id="api">
 
-This section heavily relies on the definitions from our [terminology guide](TERMINOLOGY.md). Specifically, we use the terms "reduction", "group data", "moment", and "parity" in the following.
+This section heavily relies on the definitions from our [terminology guide](TERMINOLOGY.md). Specifically, we use the terms "reduction", "sensitive features", "moment", and "parity" in the following.
 
 For all disparity mitigation methods algorithm-specific parameters are passed to the constructor. The methods to fit a mitigator and predict values with the resulting model shall resemble the APIs used by scikit-learn as much as possible for the sake of ease of use. Any deviations are noted below.
 
 ### Reductions
 
-Reductions require an estimator to be passed that implements the `fit` method with the `sample_weight` argument. The constraints for reductions are all moments (see `fairlearn.reductions.moments`) passed as instances of classes inheriting from `Moment`. Moments are vector functions that we use to formalize our constraints. The moment objects need to be passed as constraints to the constructor of a reduction technique, which internally uses the constraints to solve the optimization problem.
+Reductions require an estimator to be passed that implements the `fit` method with the `sample_weight` argument. The constraints for reductions are all moments (see `fairlearn.reductions`) passed as instances of classes inheriting from `Moment`. Moments are vector functions that we use to formalize our constraints. The moment objects need to be passed as constraints to the constructor of a reduction technique, which internally uses the constraints to solve the optimization problem.
 
 ```python
 constraints = Moment()
-reduction = Reduction(estimator, objective=objective, constraints=constraints, use_predict_proba=False, **kwargs)
+reduction = Reduction(estimator, objective=objective, constraints=constraints, **kwargs)
 ```
 
-Reduction-based disparity mitigation algorithms (such as the ones under `fairlearn.reductions`) provide `fit`, `predict`, and `predict_proba` methods with the following signatures:
+Reduction-based disparity mitigation algorithms (such as the ones under `fairlearn.reductions`) provide `fit`, `predict`, and `_pmf_predict` methods with the following signatures:
```

Comment:
Why are we mentioning `_pmf_predict` at all? Sorry if I've missed another discussion, but by putting it in the docs, we are sort of promising to support it.

Reply:
It had predict_proba there before, so I just went and replaced it. Better be consistent with the code than mention stuff we don't even have. If you want to remove it I am not opposed to that.

Reply:
Yes - I was surprised it hadn't been removed entirely, and was wondering if I'd missed a discussion